app/models/application.rb

```diff
@@ -125,6 +125,10 @@ def sufficient_votes?
     enough_yes || !few_nos
   end
 
+  def age
+    Time.now - submitted_at
```

Comment:
It is best to move logic out of the templates and into the code here (among other things, so that it can be tested), so I would change this to return the age in days, instead of just a difference between the two dates.

Also, instead of using your own logic to compute the days, you could use `in_days` on a Rails Duration to get the number that you want to display in the UI. Documentation here: https://api.rubyonrails.org/classes/ActiveSupport/Duration.html#method-i-in_days

Reply:
in_days doesn't seem to work in the rails console at all. I get an undefined method error.

Reply:
in_days does not work in the template either. Is there anything I need to do to activate it?

Reply:
It's fine, you don't need to convert it to `ActiveSupport::Duration` if that doesn't work. The important part is to get the math/logic out of the view, and to add clarity, so e.g. if you leave this method as it is now, consider renaming it to `age_secs` (assuming secs is really what it is returning) or at least adding a comment indicating what is this returning.